birdhouse/config/solr/config/canarie-api/canarie_api_monitoring.py.template

```diff
@@ -1,8 +1,39 @@
-SERVICES['node']['monitoring']['Solr'] = {
-    'request': {
-        'url': 'http://${PAVICS_FQDN}:8983/solr/birdhouse/select?q=CMIP5&fq=model:MPI-ESM-MR&fq=experiment:rcp45&fq=variable:tasmax&fq=institute:MPI-M&fq=frequency:mon&wt=json'
+SERVICES['Solr'] = {
+    'info': {
+        'name': 'Solr',
+        'synopsis': (
+            'SOLR is a search platform part of the Apache Lucene project. '
+            'It is used in this project for its faceted search capability. '
+            'Search queries are relayed from the UI or WPS processes to the SOLR database, '
+            'which returns a json file with the links to matching files.'
+        ),
+        'version': "${SOLR_VERSION}",
+        'institution': 'Ouranos',
+        'releaseTime': get_release_time_from_repo_tag("docker", "${SOLR_DOCKER}", "${SOLR_VERSION}"),
+        'researchSubject': 'Climatology',
+        'supportEmail': '${SUPPORT_EMAIL}',
+        'category': 'Data Manipulation',
+        'tags': ['Indexation', 'Search']
     },
-    'response': {
-        'text': '.*catalog_url\":\".+/testdata/flyingpigeon/cmip5.*/catalog.xml.*'
+    'stats': {
+        'method': '.*',
+        'route': '/solr/.*'
+    },
+    'redirect': {
+        'doc': 'https://ouranosinc.github.io/pavics-sdi/arch/backend.html#indexation',
+        'releasenotes': 'https://github.com/Ouranosinc/PAVICS/tags',
+        'support': 'https://github.com/Ouranosinc/PAVICS/issues',
+        'source': 'https://github.com/Ouranosinc/PAVICS/tree/master/birdhouse/docker/solr',
+        'tryme': 'http://${PAVICS_FQDN}:8983/solr/',
+        'licence': 'https://github.com/bird-house/finch/blob/master/LICENSE.txt',
+        'provenance': 'https://ouranosinc.github.io/pavics-sdi/arch/backend.html#indexation'
+    },
+    "monitoring": {
+        'Solr': {
+            'request': {
+                # FIXME: remove port by design (https://github.com/bird-house/birdhouse-deploy/issues/222)
+                'url': 'http://${PAVICS_FQDN}:8983/solr/birdhouse/select'
```

Comment:
I would prefer not to have it check for a specific file. Just because the file is added in bootstrap-instance-for-testsuite doesn't mean that a deployment of PAVICS will necessarily have added that data. I don't want canarie failing because we haven't set things up exactly as the test suite expects (when we're not running tests)

Reply:
The URL I posted above is directly from one of the builds from this PR. See the `response.numFound == 0`. 
Most probably the crawler did not complete its operation yet, or the test data is not inserted in those builds.
Either way, this regex mismatch causes the initial monitoring to loop indefinitely and never run tests.

Reply:
> It must have worked before, we have had many passing builds in the pipeline.

Maybe before monitoring was introduced in the CI to check that the instance was ready, so it silently failed?

Reply:
I agree with @mishaschwartz 
Instances are free to extend further the number of items monitored for more reporting/robustness, but we shouldn't include a test requirement on the core configuration of the service. It could be a `optional-component/testdata-canarie-monitoring`.

Reply:
So here it's true it breaks the "it just works out of the box" experience when a system is not properly configured for the testsuite.  Someone just want to try out PAVICS probably do not want to immediately configure for the testsuite.

So yes leave out the data query for Solr in the canarie-api.

Reply:
> Maybe before monitoring was introduced in the CI to check that the instance was ready, so it silently failed?

Since day one of the pipeline, Mathieu used that canarie-api monitoring link so make sure the system is properly provisioned before launching Jenkins testsuite on it (there is a notebook that query Solr as well).  So this monitoring is there since day one, it did not fail silently before monitoring is added.

I think this provisioning flakiness might be related to the same root cause that randomly sometime the stack do not boot up.

Reply:
Until about 1-2 months ago, it was only pinging the `/canarie/` endpoint and checking that there is "Ouranos" somewhere in the output. It did not guarantee that any service in the stack was ready for tests. The advanced monitoring of all services as "up" to resume tests was added by myself very recently. If they were ready, it was just coincidence with other docker services taking longer to boot, and proxy depending on them to restart.

The stack not booting up seems to be related to available docker-pool data usage. Whenever it's too high, they just start failing until we manually flush some items.